Bottom: 90edd5e8387d0301915bf024b1a497b5369760bf
Top:    d7150216b52d64eb5fb3cfaa3e1fdb080a4182da
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-07 11:57:48 +0000

Debugging on again...


---

diff --git a/src/polygon1.c b/src/polygon1.c
index ac4de07..17fbb56 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
